lib/python/ensembl/brc4/runnable/load_sequence_data.py

```diff
@@ -1014,6 +1014,7 @@ def name_and_id_from_seq_region_item(self, seq_region_item: dict, seq_region_map
         Throws exception if not able to get seq_region_id from "seq_region_map" and "throw_missing" is true.
         """
         #   get seq_region_id (perhaps, by using unversioned name)
+        unversion = self.param("unversion_scaffolds")
```

Comment:
we don't need this, but we need to use "try_unversion" instead of "unversion" in the "if" on line 1021

Reply:
changed "unversion" to "try_unversion" and it runs smoothly 👍

Reply:
cool